01-JS-DOM-APIs/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" >
+    <script type="text/javascript" src="javascript.js"> </script>
+    <meta charset="utf-8">
+
+    <title>JS</title>
+  </head>
+     <body onload="setTimeout(show, 1000);">
+
+    <section id="all">
+      <section>
+        <p> This is random content</p>
+        And this too.
+        <br>
+        <p id="hidden">
+          Hello world!
+        </p>
+        <input type="button" id="buttonShow" value="Show" onclick="show();" />
+      </section>
+      <hr>
+      <section id="jokeHidden">
+      </section>
+        <input type="button" value="Give me a joke" id="buttonjoke" onclick="buttonJokeWithoutPromise();"/>
+        <hr>
+        <section id="jokeHidden2">
+        </section>
+        <input type="button" value="Give me a joke with promise" id="button2" onclick="buttonJokeWithPromise();"/>
+      <hr>
+      <ul id="list"></ul>
+      <label>Enter your parameters</label>
+      <input type="text"  id="repoText"/>
```

Comment:
**Minor:** as a good practice, when you have text inputs in your forms, you should add a placeholder attribute, or a label to let the user know what's that input about